### DISCUSSION TITLE
Netbox with Redis Sentinel - Issue when calling /api/status/

### DISCUSSION THREAD

**Discussion by smutel** [netbox-community/netbox#7770] [Q&A]
Hello,

I setup Netbox with Redis Sentinel. It works correctly except for the page /api/status.
I have the message below:
```json
{"error": "No master found for 'master0'", "exception": "MasterNotFoundError", "netbox_version": "2.11.12", "python_version": "3.7.3"}
```

Should I create an issue for that ?
Thanks.

**Comment by phandc** [comment]
Hi @smutel, I got the same issue. Did you find the solution?
